lib/common/pyDriveUtil.py

```diff
@@ -12,8 +13,12 @@ class PyDriveUtil(object):
     def __init__(self, settings=None):
         if settings is None:
             settings = {"settings_file": None, "local_cred_file": "mycreds.txt"}
-        self.gauth = self.get_gauth(settings)
-        self.drive = GoogleDrive(self.gauth)
+        if os.path.exists(settings['local_cred_file']):
+            self.gauth = self.get_gauth(settings)
```

Comment:
`self.gauth` is not used elsewhere, I think you can use gauth instead like the refactor you made in:
https://github.com/Mozilla-TWQA/Hasal/pull/654/files#diff-9a13e7681f813c5a68e0ed49a6c4935fR89